src/sentry/tasks/post_process.py

```diff
@@ -564,6 +445,160 @@ def process_snoozes(group):
     return False
 
 
+def process_event(cache_key, data, group_id):
```

Comment:
Looks like `cache_key` is unused

Reply:
good catch that was a regression from an earlier fix